dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/upgrade/UpgradeDao.java

```diff
@@ -214,7 +214,6 @@ private void runInitDDL(String initSqlPath) {
         if (StringUtils.isEmpty(rootDir)) {
             throw new RuntimeException("Environment variable user.dir not found");
         }
-        //String mysqlSQLFilePath = rootDir + "/sql/create/release-1.0.0_schema/mysql/dolphinscheduler_ddl.sql";
         String mysqlSQLFilePath = rootDir + initSqlPath + "dolphinscheduler_ddl.sql";
```

Comment:
Since we move the sql file from `${projectRoot}/sql/xx` to classpath, do we need to change the `mysqlSQLFilePath` here?

Reply:
> Since we move the sql file from `${projectRoot}/sql/xx` to classpath, do we need to change the `mysqlSQLFilePath` here?

It's not a must because `rootDir` can be specified by users, but I can change the default value to match this PR's changes

Reply:
OK, seems to be no need because the `sql` folder is still in the root folder after packaged @ruanwenjun

Reply:
Ok.